src/site/stages/build/process-cms-exports/index.js

```diff
@@ -164,6 +166,25 @@ const entityAssemblerFactory = contentDir => {
    *                    with the body of the referenced entities.
    */
   const assembleEntityTree = (entity, ancestors = [], parentFieldName = '') => {
+    // Use the automatic transformer process for certain content models
+    // This is a test. Eventually,
```

Comment:
I'm guessing that eventually the `transformFields` will be the default and only way that we do this?